ntoskrnl/se/accesschk.c

```diff
@@ -486,7 +394,7 @@ SepAccessCheck(
     BOOLEAN Defaulted;
     NTSTATUS Status;
     PACCESS_TOKEN Token = NULL;
-    PACCESS_CHECK_RIGHTS AccessCheckRights = NULL;
+    ACCESS_CHECK_RIGHTS AccessCheckRights = {0};
```

Comment:
In this case it's probably not going to go towards usermode, but in case it ever is:
This can leak stack data if there is padding in the struct, because the padding is not zero'd out.